src/test/java/com/redhat/cloud/policies/app/RestApiTest.java

```diff
@@ -41,6 +41,7 @@
 import javax.json.bind.JsonbBuilder;
 
 import io.restassured.response.Response;
+import org.jetbrains.annotations.NotNull;
```

Comment:
`jetbrains` ?
Wondering if you are using the desired annotation.

Reply:
Nope.